okapi-core/src/main/java/org/folio/okapi/bean/ModuleInstance.java

```diff
@@ -86,4 +88,20 @@ public ModuleInstance withRetry() {
     this.withRetry = true;
     return this;
   }
+
+  public String getxOkapiUserId() {
+    return xokapiUserid;
+  }
+
+  public void setxOkapiUserId(String xokapiUserid) {
+    this.xokapiUserid = xokapiUserid;
+  }
+
+  public String getxOkapiPermissions() {
+    return xokapiPermissions;
+  }
+
+  public void setxOkapiPermissions(String xokapiPermissions) {
+    this.xokapiPermissions = xokapiPermissions;
+  }
```

Comment:
The lower x looks odd but it is required by Google Code Style.
Can we remove "xOkapi" from the name?

Reply:
renamed them.